src/app/_components/Social.tsx

```diff
@@ -1,21 +1,35 @@
-import { socialIcons, socialLinks, SocialIconKey } from '@/utils/socialConfig'
+import clsx from 'clsx'
+
+import { Icon } from '@/components/Icon'
+
+import { socialLinksWithIcons } from '@/utils/socialConfig'
+
+const touchTargetSpacing = 2
+const touchTargetClass = `p-${touchTargetSpacing}`
+const touchTargetOffsetClass = `-m-${touchTargetSpacing} sm:mx-0`
 
 export function Social() {
   return (
-    <ul className="flex flex-wrap items-center justify-between gap-4 px-3">
-      {Object.entries(socialLinks).map(([key, { label, href }]) => {
-        const IconComponent = socialIcons[key as SocialIconKey]
-
+    <ul
+      className={clsx(
+        'flex flex-wrap items-center justify-between gap-4 sm:px-8',
+        touchTargetOffsetClass,
+      )}
+    >
+      {socialLinksWithIcons.map(({ label, href, icon }, key) => {
         return (
-          <li key={key}>
+          <li key={key} className="inline-flex">
             <a
+              aria-label={`${label} (opens in new window)`}
               href={href}
               target="_blank"
               rel="noopener noreferrer"
-              className="outline-white hover:text-brand-300 focus:outline-2"
-              aria-label={`${label} (opens in new window)`}
+              className={clsx(
+                'text-brand-100 outline-white hover:text-brand-500 focus:outline-2',
```

Comment:
Hover seems to have a lighter color in the designs,` text-brand-400`?
![Screenshot 2024-04-23 at 08 09 26](https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/50910606/347e7f39-508d-401c-8eda-bd865a476f20)
![Screenshot 2024-04-23 at 08 09 34](https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/50910606/a31de2b8-b873-443e-92ca-ed4c42e22902)